trace/MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include README.rst LICENSE
-global-include *.json
+recursive-include unit_tests *
```

Comment:
Is it `tests`? We're really inconsistent about the naming here. 

Either way, we do include unit tests in our source distributions (but not our wheels). Specifically for the benefit of downstream packagers (e.g., linux distros).